sourcegraph/about-replatform
# Add uninstall page

Closes #97 by adding uninstall page. Also removes some unused styles.

## To Test
- Check out the page at `/uninstall` and make sure form loads and looks correct.
- Check static builds
